reference/extensions/binary_compatibility.rst

```diff
@@ -0,0 +1,24 @@
+.. _reference_extensions_binary_compatibility:
+
+Binary compatibility
+--------------------
+
+This plugin, located in the cache ``extensions/plugins/compatibility/compatibility.py`` allows defining custom
+rules for the binary compatibility of packages accross settings and options.
+It has some built-in logic implemented, but can be customized.
+
+
+.. code-block:: python
+
+    def compatibility(conanfile):
+        result = []
+        if conanfile.settings.build_type == "Debug":
+            result.append({"settings": [("build_type", "Release")]})
+        return result
```

Comment:
will it work in conjunction to `built-in` logic, or completely override it? if second, how to invoke built-in logic from my own `compatibility.py` plug-in?